.gitignore

```diff
@@ -0,0 +1 @@
+/home/subashj/hep_bham/dune/dtp_v1.2_test/test_ipbus_1.10/df-work/src/dtp-controls/.gitignore
```

Comment:
Custom path in gitignore???

Reply:
Used a symlink was a bad idea. Shall fix this tomorrow.

Reply:
fixed bbe29ec.